src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -209,7 +209,7 @@ public static bool IsWindowsConsumption(this IEnvironment environment)
         /// <returns><see cref="true"/> if running in one of our Consumption SKUs, false otherwise.</returns>
         public static bool IsConsumptionSku(this IEnvironment environment)
         {
-            return IsWindowsConsumption(environment) || IsAnyLinuxConsumption(environment) || IsFlexConsumptionSku(environment);
+            return IsWindowsConsumption(environment) || IsAnyLinuxConsumption(environment) || IsFlexConsumptionSku(environment) || IsConsumptionV1RunningOnLegion(environment);
```

Comment:
I would think that IsAnyLinuxConsumption would return true for cv1 on Legion without needing to add a new check here?